### PR TITLE
Make waitForPageLoad work without jQuery, disable HTML validation for API.

### PR DESCRIPTION
**Pull request by EreMaijala**

### CODE REVIEW

File: module/VuFind/src/VuFindTest/Integration/MinkTestCase.php [hunk 1]
```diff
@@ -741,38 +741,45 @@ protected function waitForPageLoad(
...
-            $timeout,
-            "typeof $ !== 'undefined' && $.active === 0"
-        );
+        // Check for jQuery (if not loaded, some checks will be skipped):
+        $jQuery = $session->wait($timeout, "typeof $ !== 'undefined'");
+        if ($jQuery) {
+            // Wait for any AJAX requests to complete:
+            $session->wait(
+                $timeout,
+                "typeof $ !== 'undefined' && $.active === 0"
```
Comment (demiankatz) [inline module/VuFind/src/VuFindTest/Integration/MinkTestCase.php]: Can you remove the `typeof` check here since you already did it above?

**Review by demiankatz — CHANGES_REQUESTED** [review]
Thanks, @EreMaijala -- one small possible suggestion (which you're free to ignore if there's a reason to).

Beyond that, when I actually run this with HTML validation on, I'm seeing a large number of errors. I assume this is what you're seeing too -- mostly repeated comments about attributes on the search box, but a handful of other scattered issues too. Just wanted to make sure we're seeing more or less the same thing before I approve and merge this.

File: module/VuFind/src/VuFindTest/Integration/MinkTestCase.php (hunk 1 above)
Comment (EreMaijala) [reply module/VuFind/src/VuFindTest/Integration/MinkTestCase.php]: Done.

**Comment by EreMaijala** [comment]
@demiankatz Yes, I'm seeing the same errors.

**Review by demiankatz — APPROVED** [review]
Thanks, @EreMaijala!

**Pull request merged**